arduino/builder/sketch.go

```diff
@@ -133,8 +134,8 @@ func LoadSketch(sketchPath, buildPath string) (*sketch.Sketch, error) {
 	return sketch.New(sketchFolder, mainSketchFile, buildPath, files)
 }
 
-// MergeSketchSources merges all the source files included in a sketch
-func MergeSketchSources(sketch *sketch.Sketch) (int, string) {
+// SketchMergeSources merges all the source files included in a sketch
+func SketchMergeSources(sketch *sketch.Sketch) (int, string) {
```

Comment:
I'm wondering if these functions (`SketchMergeSources`, `SketchCopyAdditionalFiles`, etc.) may become methods of `sketch.Sketch`?